lib/util.coffee

```diff
@@ -78,7 +79,7 @@ module.exports = Util =
           apd = stackpath.split(delimiter).concat apd
         if sbd
           apd.unshift sandbox
-    env.PATH = joinPath(apd)
+    env.PATH = joinPath(apd) if env.PATH == ''
```

Comment:
I’m essentially treating the configs as fallbacks now, but maybe we’d be better off removing them altogether. Thoughts?